packages/story-editor/src/components/canvas/karma/rightClickMenu.karma.js

```diff
@@ -46,6 +51,118 @@ describe('Right Click Menu integration', () => {
     fixture.restore();
   });
 
+  function rightClickMenu() {
+    return fixture.screen.getByRole('group', {
+      name: 'Context Menu for the selected element',
+    });
+  }
+
+  function sendBackward() {
+    return fixture.screen.getByRole('button', {
+      name: /^Send Backward/i,
+    });
+  }
+
+  function sendToBack() {
+    return fixture.screen.getByRole('button', {
+      name: /^Send to Back/i,
+    });
+  }
```

Comment:
The right click menu is in a portal, so these were moved to search on the screen so that the tests would pass.